README.md

```diff
@@ -1,14 +1,54 @@
+
+### Highlight matched text
+The high-performance Blazor AutoComplete control lets you configure different visual appearances for the matching text of each suggestion.
+
+## Related links
+[Learn More about Blazor AutoComplete](https://www.syncfusion.com/blazor-components/blazor-autocomplete?utm_source=github&utm_medium=listing&utm_campaign=blazor-autocomplete-github-samples)
+
+[Documentation](https://blazor.syncfusion.com/documentation/autocomplete/getting-started?utm_source=github&utm_medium=listing&utm_campaign=blazor-autocomplete-github-samples)
+
+[Download Free Trial](https://www.syncfusion.com/downloads/blazor/confirm?utm_source=github&utm_medium=listing&utm_campaign=blazor-autocomplete-github-samples)
+
+[Release History](https://blazor.syncfusion.com/documentation/release-notes/19.4.38?type=all?utm_source=github&utm_medium=listing&utm_campaign=blazor-autocomplete-github-samples)
+
+[Online Demos](https://github.com/syncfusion/blazor-samples?utm_source=github&utm_medium=listing&utm_campaign=blazor-autocomplete-github-samples)
+
+[Contact Support](https://www.syncfusion.com/support/directtrac/incidents/newincident/?utm_source=github&utm_medium=listing&utm_campaign=blazor-autocomplete-github-samples)
+
+[Suggest a feature or report a bug](https://www.syncfusion.com/feedback/blazor?utm_source=github&utm_medium=listing&utm_campaign=blazor-autocomplete-github-samples)
+
+[Community Forums](https://www.syncfusion.com/forums?utm_source=github&utm_medium=listing&utm_campaign=blazor-autocomplete-github-samples)
+
+[Knowledge base](https://www.syncfusion.com/kb?utm_source=github&utm_medium=listing&utm_campaign=blazor-autocomplete-github-samples)
+
+[Pricing](https://www.syncfusion.com/sales/teamlicense?utm_source=github&utm_medium=listing&utm_campaign=blazor-autocomplete-github-samples)
+
+## About Syncfusion Blazor UI Controls
+Syncfusion's [Blazor UI Controls](https://www.syncfusion.com/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-autocomplete-github-samples) library is the only suite that you will ever need to build an application since it contains over 100 high-performance, lightweight, modular, and responsive UI Controls in a single package. In addition to AutoComplete component, we provide popular Blazor Controls such as [DataGrid](https://www.syncfusion.com/blazor-components/blazor-datagrid?utm_source=github&utm_medium=listing&utm_campaign=blazor-autocomplete-github-samples), [Charts](https://www.syncfusion.com/blazor-components/blazor-charts?utm_source=github&utm_medium=listing&utm_campaign=blazor-autocomplete-github-samples), [Diagram](https://www.syncfusion.com/blazor-components/blazor-diagram?utm_source=github&utm_medium=listing&utm_campaign=blazor-autocomplete-github-samples), and [Scheduler](https://www.syncfusion.com/blazor-components/blazor-scheduler?utm_source=github&utm_medium=listing&utm_campaign=blazor-autocomplete-github-samples).
+
```

Comment:
Check this and change
![image](https://github.com/SyncfusionExamples/blazor-auto-complete-getting-started-example/assets/95272441/0f0376ce-5cba-45e4-995d-072b87ad9d94)